src/pages/u/[handle].tsx

```diff
@@ -4,14 +4,51 @@ import withAccessType from '@/lib/hoc/withAccessType';
 import { CookieService, PermissionService } from '@/lib/services';
 import type { PublicProfile } from '@/lib/types/apiResponses';
 import { CookieType } from '@/lib/types/enums';
+import defaultProfilePicture from '@/public/assets/profile-pics/adorable0.png';
+import styles from '@/styles/pages/u.module.scss';
+import Image from 'next/image';
 import type { GetServerSideProps } from 'next/types';
 
 interface UserProfilePageProps {
   user: PublicProfile;
 }
 
 const UserProfilePage = ({ user }: UserProfilePageProps) => {
-  return <pre>{JSON.stringify(user, null, 2)}</pre>;
+  // return <pre>{JSON.stringify(user, null, 2)}</pre>;
+  return (
+    <div className={styles.profilePage}>
+      <div className={styles.cardWrapper}>
+        <div className={styles.banner} />
+        <div className={styles.profileCard}>
+          <div className={styles.cardHalf}>
+            <div className={styles.horizontal}>
+              <Image
+                src={user.profilePicture ?? defaultProfilePicture}
+                alt="Profile Picture"
+                className={styles.profilePic}
+              />
+              <div style={{ marginLeft: '1rem' }}>
+                <h1>{`${user.firstName} ${user.lastName}`}</h1>
+                <h3>@{user.handle}</h3>
+                <h2>Rank Here</h2>
+              </div>
+            </div>
+            <div className={styles.iconBox}>icon icon icon icon</div>
+          </div>
+          <div className={styles.cardHalf}>
+            <h2>Points</h2>
+            <h2>Class of {user.graduationYear}</h2>
+            <h2>{user.major}</h2>
+          </div>
+        </div>
+      </div>
+
+      <div className={styles.about}>
+        <h1>About me</h1>
+        <p>{user.bio ?? <i>Nothing here...</i>}</p>
```

Comment:
The `??` operator is for nullish coalescing and it's a good idea when we're trying to check if the value is completely missing (null or undefined) and use a default value but in this case, we actually don't want that because it may be an empty string, which wouldn't correctly show the default.

![Screenshot 2023-05-08 at 10 56 29 AM](https://user-images.githubusercontent.com/33165426/236896059-88786f58-4687-4ca2-927e-000d47fd62e2.jpg)

Reply:
fixed in Clean up code 1